enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
+1. The policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of `InProgress`.
+1. All the clusters in the `prod` group become compliant. Their `rolloutStatus` is set to `Succeeded`. The root policy
+   `status.rolloutStatus` value is `Succeeded`.
+
+#### New Policy Failed
+
+##### First Rollout
+
+1. The policy applies to all clusters in the `dev` group. These clusters have the `rolloutStatus` of `InProgress`. The
+   remaining clusters have the `rolloutStatus` of `ToApply`. The root policy `status.rolloutStatus` value is
+   `InProgress`.
+1. All the clusters in the `dev` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of `InProgress`.
+1. One cluster in the `stage` group stays noncompliant for the duration of the `spec.rolloutStrategy.progressDeadline`
+   of 10 minutes. Its `rolloutStatus` is set to `Failed`. The root policy `status.rolloutStatus` value is `Failed`. If
+   `spec.rolloutStrategy.progressDeadline` was not set, it would have waited indefinitely and never failed.
+
+##### Second Rollout
+
+1. The root policy is updated.
+1. The updated policy applies to all clusters in the `dev` group. These clusters have the `rolloutStatus` of
+   `InProgress`. The remaining clusters have the `rolloutStatus` of `ToApply` and keep whatever policy version, if any,
+   from the failed rollout. The root policy `status.rolloutStatus` value is `InProgress`.
+1. All the clusters in the `dev` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `prod` group become compliant. Their `rolloutStatus` is set to `Succeeded`. The root policy
+   `status.rolloutStatus` value is `Succeeded`.
+
+#### Updated Policy Succeeded
+
+1. The root policy is updated.
+1. The updated policy applies to all clusters in the `dev` group. These clusters have the `rolloutStatus` of
+   `InProgress`. The remaining clusters have the `rolloutStatus` of `ToApply` and keep the previous policy version. The
+   root policy `status.rolloutStatus` value is `InProgress`.
+1. All the clusters in the `dev` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The updated policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of
```

Comment:
Individual status of group only appears on `kind: rollout?`

Reply:
We don't have statuses per group but only rollout statuses per cluster and a summary rollout status to reflect all clusters.